Src/BootCamp.Chapter/CleanTheFile.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    static class CleanTheFile
+    {
+        public static void Clean(string file, string outputFile)
+        {
+            if (Testers.IsThisStringValid(file) || Testers.IsThisStringValid(outputFile))
+            {
+                throw new ArgumentException();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use a constructor overloads that allows a more meaningful exception message to be provided.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)